_1327/settings.py

```diff
@@ -51,6 +51,7 @@
 	'reversion',
 	'guardian',
 	'polymorphic',
+	'django_extensions',
```

Comment:
are you sure that the system works without the test requirements installed if you add this app to the installed apps here?